client/modules/core/csvjsonparse/rowInfoComponent/rowInfo.component.ts

```diff
@@ -0,0 +1,19 @@
+import {
+    Component,
+    Input
+} from '@angular/core';
+import {
+    AssignCategoryComponent
+} from './assignCategoryComponent/assignCategory.component';
+import template from './rowInfo.html';
+
+@Component({
+    selector: '[other]',
+    template,
+    directives: [AssignCategoryComponent]
+})
+
+export class RowInfoComponent {
+    @Input() row: string[];
```

Comment:
instead of this being a array of string. make it a json object with proper type checking. 

create a row.ts model file so that we can have proper type checking